__tests__/task.test.ts

```diff
@@ -0,0 +1,72 @@
+import supertest from 'supertest';
+import dotenv from 'dotenv';
+import {
+  afterAll, beforeEach, describe, test, expect,
+} from '@jest/globals';

```

Comment:
Jest puts each of these methods into the global environment so u don't have to import anything to use them.

Reply:
ok, thanks.